p8e-migration/sql/V69__Add_UUID_Key_To_Affiliate.sql

```diff
@@ -1,2 +1,2 @@
 ALTER TABLE affiliate ADD COLUMN signing_key_uuid UUID;
-ALTER TABLE p8e.affiliate ADD COLUMN key_provider_type TEXT NOT NULL DEFAULT 'DATABASE';
+ALTER TABLE affiliate ADD COLUMN key_provider_type TEXT NOT NULL DEFAULT 'DATABASE';
```

Comment:
I think this can't change now since it was already applied in figure test and now the file hash won't match.

Reply:
what is the work around for this then? test doesn't know what the `p8e` schema is

Reply:
So this migration hasn't been applied in test? How does the branch code run without it?

Reply:
misunderstood the timeline, this has already been applied to test, after removing of `p8e.` and is currently working fine.

Reply:
👍